sap/rest/gcts/simple.py

```diff
@@ -57,7 +74,10 @@ def clone(connection, url, name, vsid='6IT', start_dir='src/', vcs_token=None, e
         repo.wipe_data()
 
     if not repo.is_cloned:
-        repo.clone()
+        try:
+            repo.clone()
+        except HTTPRequestError as exc:
+            _wait_for_clone(repo, wait_for_ready, exc)
```

Comment:
We need to be more verbose. We must let users know that we are now waiting because the clone http request returned 500.

Also a message announcing the feature in the case the waiting is not configured would not be a bad idea. However, then you need move the loggin and progress reporting from sap/rest/gcts/simple.py to sap/cli/gcts.py because that part is responsible for communication with users.